### PR TITLE
Feature CPD-1018 & CPD-1031: Historical task data and task stats

### PR DESCRIPTION
**Pull request by izzy64**
#  #

## 🗣 Description ##

Front-end Changes: https://github.com/cisagov/con-pca-web/pull/432

Jira Tickets:
https://cset.atlassian.net/browse/CPD-1018
https://cset.atlassian.net/browse/CPD-1031

## 💭 Motivation and context ##

Requests from the client.

## 🧪 Testing ##

Tested Locally.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] All new and existing tests pass.

### CODE REVIEW

File: etc/entrypoint.sh [hunk 1]
```diff
@@ -4,6 +4,8 @@ echo "Updating geoip database"
 echo "AccountID $MAXMIND_USER_ID" >> /usr/local/etc/GeoIP.conf
 echo "LicenseKey $MAXMIND_LICENSE_KEY" >> /usr/local/etc/GeoIP.conf
 geoipupdate
+echo "Make updates to the database"
+flask transform-data
```
Comment (itsmostafa) [inline etc/entrypoint.sh]: can we call this as a function just like the other initialization functions in main.py

File: src/api/main.py [hunk 2]
```diff
@@ -196,6 +197,7 @@
     initialize_templates()
     initialize_nonhumans()
     populate_stakeholder_shortname()
+    populate_cycle_tasks
```
Comment (itsmostafa) [inline src/api/main.py]: should this be a function call?

**Review by itsmostafa — COMMENTED** [review]
everything else looks good!

File: src/api/main.py (hunk 2 above)
Comment (izzy64) [reply src/api/main.py]: Yes! My mistake. Resolved with [f6b33a6](https://github.com/cisagov/con-pca-api/pull/839/commits/f6b33a66119c06a41763e8a4f9df2a851c9a3ca3)

File: etc/entrypoint.sh (hunk 1 above)
Comment (izzy64) [reply etc/entrypoint.sh]: Yep, just tested and when I correctly call the function in main.py, this command is not necessary. Resolved with [f6b33a6](https://github.com/cisagov/con-pca-api/pull/839/commits/f6b33a66119c06a41763e8a4f9df2a851c9a3ca3)

**Review by itsmostafa — APPROVED** [review]

**Pull request merged**